modules/tracktion_engine/playback/devices/tracktion_InputDevice.h

```diff
@@ -262,6 +262,8 @@ class InputDeviceInstance   : protected juce::ValueTree::Listener
 
         /** Override this to receive MIDI input from the device if it has any. */
         virtual void handleIncomingMidiMessage (const juce::MidiMessage&) {}
+
+        virtual void discardRecordings() {}
```

Comment:
This needs a comment describing what the method should do (see the other virtual methods.
Explain this is called when a recording is aborted so listeners should trash their temp data.